release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -170,6 +170,21 @@ Previously, you could not specify the subdomain of a route, and the `spec.host`
 
 You can use this enhancement to simplify sharding by enabling a route to have multiple, distinct host names determined by each router deployment that exposes the route.
 
+[id="ocp-4-11-external-dns-operator"]
+==== External DNS Operator
+
+In {product-title} 4.11, the External DNS Operator is available for AWS Route53, Azure DNS, GCP DNS, and Infoblox as a General Availability (GA). The External DNS Operator is still a Technology Preview (TP) for BlueCat and AWS Route53 on GovCloud.
```

Comment:
This should be rephrased. The Operator is not a General Availability, but it has reached GA status. Likewise, it is not a Technology Preview, but it is in TP status.

Reply:
Addressed